### PR TITLE
Improve work

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: commands/work/work.go [hunk 1]
```diff
@@ -26,50 +21,23 @@ func (cmd) Summary() string {
...
-	// Setup logger
-	var level string
-	if l := args["--logging-level"]; l != nil {
-		level = l.(string)
-	}
-	logger, err := runtime.CreateLogger(level)
+	config, err := worker.LoadConfigFile(args["<config.yml>"].(string))
 	if err != nil {
-		os.Stderr.WriteString(err.Error())
+		fmt.Println(err)
```
Comment (walac) [inline commands/work/work.go]: nit: print errors to stderr

```go
fmt.Fprintln(os.Stderr, err)
```

File: commands/work/work.go [hunk 2]
```diff
@@ -26,50 +21,23 @@ func (cmd) Summary() string {
...
-	if err != nil {
-		logger.Error("Failed to open configFile, error: ", err)
-		return false
-	}
-	var config interface{}
-	err = yaml.Unmarshal(configFile, &config)
+	w, err := worker.New(config, nil)
 	if err != nil {
-		logger.Error("Failed to parse configFile, error: ", err)
+		fmt.Println(err)
```
Comment (walac) [inline commands/work/work.go]: nit: print errors to stderr

File: worker/config.go [hunk 3]
```diff
@@ -175,6 +200,60 @@ func ConfigSchema() schematypes.Object {
...
 			"workerGroup",
 			"workerId",
+			"temporaryFolder",
+			"logLevel",
 		},
 	}
 }
+
+// LoadConfigFile will load configuration options from a YAML file and validate
+// against the config file schema, returning an error messages explaining what
```
Comment (walac) [inline worker/config.go]: s/messages/message/

**Comment by walac** [comment]
Just a few small nits. After fixing them, feel free to merge it.

File: commands/work/work.go (hunk 1 above)
Comment (jonasfj) [inline commands/work/work.go]: hmm... I want to do this... but right now I haven't been doing this consistently. Let's fix it separately and make some utility for logging to stderr, this is certainly not the only place we do this wrong.

Notably the qemu-build and run commands...

**Pull request merged**